### ISSUE TITLE
bundle -o doesn't work?

### ISSUE THREAD

**Issue by fourpastmidnight** [ojkelly/yarn.build#96]
## Describe the bug

I ran the command `yarn bundle ../../release` from the folder of a workspace which I wanted to bundle. The resulting `bundle.zip` file was placed in the directory in which the `yarn bundle` command was executed.

## To Reproduce

```shell
PS C:\My Project> cd packages/project-to-bundle
PS C:\My Project\packages\project-to-bundle> yarn bundle -o ../../release
# ... Some time later....
PS C:\My Project\packages\project-to-bundle> ls
# .... lots of dirs and files...
bundle.zip
PS C:\My Project\packages\project-to-bundle>
```

# Expected behavior

Since the default name of the zip file if none is provided by the `-a` option is `bundle.zip`, I expected the bundle to be placed in `../../release` as specified via `-o`, i.e. the file path of the bundle should be, relative to the location where I executed the bundle command, `../../release/bundle.zip`, but instead, it was `./bundle.zip`.

**Desktop :**
 - OS: Windows 10, no WSL

**Comment by ojkelly** [comment]
Hmm yep. Just checked the script, and it looks like somewhere along the way it got wiped out. The argument is still there, but it's not being used.

This is something that we can add, though I'm not sure on the timeframe of it yet (have a few competing priorities). If you wanted to give it a go a PR would be welcome.

It should be a matter of checking if `ouputDirectory` is set and updating the variable here to use it
`      const outputArchive = ppath.join(originalCwd, this.archiveName);`
https://github.com/ojkelly/yarn.build/blob/trunk/packages/plugins/plugin-build/src/commands/bundle/index.ts#L143

**Comment by lodmfjord** [comment]
I tested this and it works now. Close?

**Comment by ojkelly** [comment]
Yep this was fixed recently, but I must have missed closing this ticket.
